app/os_windows.go

```diff
@@ -108,6 +112,9 @@ func newWindow(win *callbacks, options []Option) {
 		defer winMap.Delete(w.hwnd)
 		w.Configure(options)
 		w.ProcessEvent(Win32ViewEvent{HWND: uintptr(w.hwnd)})
+		if u := startupURI(); u != "" {
+			w.onOpenURI(u)
+		}
 		windows.SetForegroundWindow(w.hwnd)
```

Comment:
This doesn't seem right. It triggers for every new window, right? I think `startupURI` should be checked when `app.Events` is called.